src/main/resources/io/jenkins/plugins/csp/ContentSecurityPolicyManagementLink/index.jelly

```diff
@@ -71,7 +71,14 @@ THE SOFTWARE.
                                     <j:set var="plugin" value="${record.contextPlugin}"/>
                                     <j:choose>
                                         <j:when test="${plugin == null}">
-                                            ${%contextWithoutPlugin(record.contextClassName)}
+                                            <j:choose>
+                                                <j:when test="${empty(record.contextClassName)}">
+                                                    <em>(context class unknown)</em>
```

Comment:
In _Elements of Typographic Style,_ Robert Bringhurst writes that parentheses should never be italicized.

```suggestion
                                                    (<em>context class unknown</em>)
```

![258003595-4b464c76-6bf0-4cda-a2b6-5cb420b7ba7b](https://github.com/user-attachments/assets/1a7afa10-b419-4a8d-bcca-95d818084a69)

Reply:
Interesting, thanks for the reference.

In the context of this PR I'm keeping it for consistency with _(anonymous)_ but might be a potential followup change.